_episodes/03-skimage-images.md

```diff
@@ -361,13 +367,12 @@ plt.imshow(image, cmap="gray")
 {: .language-python}
 
 The first argument to `iio.imread()` is the filename of the image.
-The second argument `mode="L"` defines the type and depth of a pixel in the
-image (e.g., an 8-bit pixel has a range of 0-255). This argument is forwarded
-to the `pillow` backend, for which mode "L" means 8-bit pixels and
-single-channel (i.e., grayscale). `pillow` is a Python imaging library; which
-backend is used by `iio.imread()` may be specified (to use `pillow`, you would
-pass this argument: `plugin="pillow"`); if unspecified, `iio.imread()`
-determines the backend to use based on the image type.
+The second argument `mode="L"` defines the datatype and range of the pixel values in the image (e.g., an 8-bit pixel has a range of 0-255). This argument is forwarded to the `pillow` backend, a Python imaging library for which mode "L" means 8-bit pixels and single-channel (i.e., grayscale). The backend used by `iio.imread()` may be specified as an optional argument: to use `pillow`, you would
+pass `plugin="pillow"`. If the backend is not specified explicitly, `iio.imread()` determines the backend to use based on the image type.
+
+> ## Loading images with `imageio`: Datatypes
+> When loading an image with `imageio`, the pixel values are stored as 8-bit integer numbers (`uint8`) that can take values in the range 0-255. However, other libraries may use different datatypes. For example, some `skimage` functions return the pixel values as `dtype('float64')`. The type and range of the pixel values are important for the colorscale when plotting, and for masking and thresholding images as we will see later in the lesson. If you are unsure about the datatype of your image array, you can inspect it with `image.dtype`. For the example above, you should find that it is `dtype('uint8')`.
```

Comment:
Aww, I don't agree with this sentence. :grimacing: 

> When loading an image with `imageio`, the pixel values are stored as 8-bit integer numbers (`uint8`) that can take values in the range 0-255.

It depends on the image being loaded... When loading *this* image with `imageio`, the pixel values are stored as 8-bit integer numbers (and the following should be phrased as a reminder, because we said just earlier that "an 8-bit pixel has a range of 0-255" -- here it sounds as if the 0-255 range consisted in some kind of additional information).

Again, the read function from `skimage.io` would also read *this* image into `dtype('uint8')` by default, it's only the grayscale option which would convert it into floating point (arguably, this implicit conversion could be considered a bug or a feature).

Also, since the word datatype / data type / data type keeps showing up, I think we might as well introduce NumPy's `dtype` properly (something I thought we could avoid earlier in this PR review, but I've changed my mind). Let's say, from the get-go, that by 'data type' we mean NumPy's `dtype` and maybe even link to the official docs: https://numpy.org/doc/stable/reference/arrays.dtypes.html